### ISSUE TITLE
FR: Multidoc record

### ISSUE THREAD

**Issue by wlynch** [eddiezane/hook#5]
Many webhook providers will send all events to a single endpoint. We should make it easy to record multiple events to the same file.

When running hook in record mode, additional events that are received should be appended to the file as a separate document, separated with `---`

e.g.

```yaml
method: POST
body: foo
---
method: GET
body: bar
```

**Comment by eddiezane** [comment]
Do you think they should be combined into the same file or separate files should be created?

e.g.

```
webhooks/github/pullrequest.yml
webhooks/github/commit.yml
```

I've been thinking of having different files to address updating/automating in the future.

**Comment by wlynch** [comment]
For the catalog we should generally prefer single hooks per file, but allowing multiple for fire is an interesting side effect.

For record though, I think that might be infeasible to enforce single files. For example, a sIngle git push might trigger mutliple ref and PR updates. The easiest thing would be to collect everything and allow users to delete/move hooks around later.

**Comment by eddiezane** [comment]
Closed via #8 and #9
